<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQ</class>
 <widget class="QMainWindow" name="QQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>141</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;

border:1px solid#000000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>5</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>经典</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>自然</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>柔和</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,50);</string>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="rootIsDecorated">
     <bool>true</bool>
    </property>
    <property name="itemsExpandable">
     <bool>true</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>添加好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>个性装扮</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>播放曲目</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开歌词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭歌词</string>
    </property>
   </widget>
   <widget class="lrc" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>9</y>
      <width>470</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>344</y>
      <width>140</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
border-image: url(:/new/prefix1/上一首.jpg);

border:1px solid#000000;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
border-image: url(:/new/prefix1/播放.jpg);
border:1px solid#000000;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
border-image: url(:/new/prefix1/下一首.jpg);

border:1px solid#000000;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
border-image: url(:/new/prefix1/stop.jpg);

border:1px solid#000000;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>好友通知</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>treeWidget</zorder>
   <zorder>listWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>radioButton_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>widget</zorder>
   <zorder>pushButton_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lrc</class>
   <extends>QWidget</extends>
   <header location="global">lrc.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
